apidoc/Titanium/Android/Activity.yml

```diff
@@ -193,6 +193,12 @@ methods:
         type: Titanium.Android.Intent
         optional: true
 
+  - name: setSupportActionBar
+    summary: Sets a toolbar instance to be used as an ActionBar.
+    parameters:
+      - name: toolbar
+        type: Titanium.UI.Android.Toolbar  
```

Comment:
I haven't described that as I should. I will add a more detailed description.
The Activity class does not handle property changes. So this method is used if you want to set a Toolbar as ActionBar after the instance is created. So technically currently this is not a setter for the supportToolbar (maybe I should assign it the toolbar reference).